components/footer.css

```diff
@@ -0,0 +1,8 @@
+footer {
+  position: fixed;
+  bottom: 0rem;
+  width: 100%;
+  background-color: #be7f6a;
+}
+
+/*there is this gap on the left side of the footer, any ideas how I could remove?*/
```

Comment:
did you set the margin of your body to 0 ? this could help.

Reply:
added `margin: -0.65rem` but this is probably not the cleanest solution.